web/pdf_page_view.js

```diff
@@ -1240,4 +1277,229 @@ class PDFPageView extends PDFPageViewBase {
   }
 }
 
-export { PDFPageView };
+/**
+ * @implements {IRenderableView}
+ */
+class PDFPageDetailView extends PDFPageViewBase {
+  constructor({ pageView }) {
+    super(pageView);
+
+    this.pageView = pageView;
+    this.renderingId = "detail" + this.id;
+
+    this.detailArea = null;
```

Comment:
Nit: Unless something "outside" of the class depends on this, can it be a private field instead?